src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.exception.RegistrationException;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.mapper.impl.response.UserResponseMapper;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.security.AuthenticationService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+    private final UserService userService;
+    private final DtoResponseMapper<UserResponseDto, User> userResponseMapper;
+
+    public AuthenticationController(
+            AuthenticationService authenticationService,
+            UserService userService, UserResponseMapper userResponseMapper) {
+        this.authenticationService = authenticationService;
+        this.userService = userService;
+        this.userResponseMapper = userResponseMapper;
+    }
+
+    @PostMapping("/register")
+    public UserResponseDto register(@RequestBody UserRequestDto userRequestDto) {

```

Comment:
This method should return `void` 
![image](https://user-images.githubusercontent.com/32339836/235703260-af195dbb-e12c-440e-b70a-dbc9879bd6f0.png)